clients/src/main/java/org/apache/kafka/common/requests/ListOffsetsRequest.java

```diff
@@ -65,7 +66,8 @@ public static Builder forReplica(short allowedVersion, int replicaId) {
         public static Builder forConsumer(boolean requireTimestamp,
                                           IsolationLevel isolationLevel,
                                           boolean requireMaxTimestamp,
-                                          boolean requireTieredStorageTimestamp) {
+                                          boolean requireTieredStorageTimestamp,
+                                          List<ListOffsetsTopic> topics) {
```

Comment:
Could you please follow the pattern? You can add a flag to pick up the min version. Current implementation does not consider the version=8 and I feel that is a kind of bug. @FrankYang0529 WDYT?

Reply:
Hi @chia7712 
I have add a `requireEarliestLocalTimestamp` parameter to `forConsumer` method, PTAL

Reply:
Yes, we should update this. Thanks @chia7712 and @frankvicky